pom.xml

```diff
@@ -58,7 +58,7 @@
         <jackson-asl.version>1.9.13</jackson-asl.version>
         <liquibase.version>3.4.1</liquibase.version>
         <liquibase-slf4j.version>1.2.1</liquibase-slf4j.version>
-        <cloudfoundry.client.version>1.3.0</cloudfoundry.client.version>
+        <cloudfoundry.client.version>1.3.1-SNAPSHOT</cloudfoundry.client.version>
```

Comment:
Do not forget to adopt release version before merge

Reply:
yes, that is why I left this with `SNAPSHOT` suffix